template/app/src/main/java/co/nimblehq/template/ui/screens/compose/HomeComposeViewModel.kt

```diff
@@ -23,16 +21,17 @@ class HomeComposeViewModel @Inject constructor(
 
     init {
         execute {
-            when (val result = useCase.execute()) {
-                is UseCaseResult.Success -> {
-                    val uiModels = result.data.toUiModels()
-                    _uiModels.emit(uiModels)
-                }
-                is UseCaseResult.Error -> {
-                    val errorMessage = result.exception.message.orEmpty()
+            showLoading()
```

Comment:
@luongvo Let's revert these changes. I saw you already updated in [this PR](https://github.com/nimblehq/android-templates/pull/292/files#diff-eb44200d9d6b03ba7a180a699c76534b141ad10297fb217d311a8fa4733a5a9d) So for this PR we can focus only the changes in `template.

Reply:
@Wadeewee Yes, but it's in the template 😂